ui/react-components/components/Location/LocationSearch.jsx

```diff
@@ -2,52 +2,80 @@ import Dropdown from "../Dropdown/Dropdown.jsx";
 import React, {useEffect, useState} from "react";
 import PropTypes from "prop-types";
 import {injectIntl} from 'react-intl';
-import {getAllByTag} from "../../api/locationApi";
+// import {getAllByTag} from "../../api/locationApi";
+import {getAllByTag} from "../../api/__mocks__/locationApi";
```

Comment:
Is it good idea to import from `__mocks__` in source code? Similar in `ProviderSearch`.

Reply:
No. that is done just for checking the output. we will update the code with real API call in both the files - LocationSearch and ProviderSearch

Reply:
> we will update the code with real API call in both the files - LocationSearch and ProviderSearch
Do you mea it needs to be changed later sometime after the PR is merged?

Reply:
No. I have removed in the last commit. And pushed the updated code

Reply:
Thanks